.github/workflows/builds.yml

```diff
@@ -279,7 +279,7 @@ jobs:
           && git config --global safe.directory "$GITHUB_WORKSPACE"
           && pip3 install histcmp==0.5.2
           && pip3 install -r Examples/Scripts/requirements.txt
-          && /usr/local/bin/download_geant4_data.sh
+          && /usr/local/bin/geant4-config --install-datasets
```

Comment:
Turns out G4 actually ships with a script that does exactly what we need here. 😅